config/examples/Two Trees/Sapphire Pro/Configuration.h

```diff
@@ -0,0 +1,2501 @@
+#define CONFIGURATION_H_VERSION 020008
+
+//===========================================================================
+//============================= Getting Started =============================
+//===========================================================================
+
+/**
+ * Here are some standard links for getting your machine calibrated:
+ *
+ * https://reprap.org/wiki/Calibration
+ * https://youtu.be/wAL9d7FgInk
+ * http://calculator.josefprusa.cz
+ * https://reprap.org/wiki/Triffid_Hunter%27s_Calibration_Guide
+ * https://www.thingiverse.com/thing:5573
+ * https://sites.google.com/site/repraplogphase/calibration-of-your-reprap
+ * https://www.thingiverse.com/thing:298812
+ */
+
+//===========================================================================
+//============================= DELTA Printer ===============================
+//===========================================================================
+// For a Delta printer start with one of the configuration files in the
+// config/examples/delta directory and customize for your machine.
+//
+
+//===========================================================================
+//============================= SCARA Printer ===============================
+//===========================================================================
+// For a SCARA printer start with the configuration files in
+// config/examples/SCARA and customize for your machine.
+//
+
+// @section info
+
+// Author info of this build printed to the host during boot and M115
+#define STRING_CONFIG_H_AUTHOR "(ils15, default config)" // Who made the changes.
+//#define CUSTOM_VERSION_FILE Version.h // Path from the root directory (no quotes)
+
+/**
+ * *** VENDORS PLEASE READ ***
+ *
+ * Marlin allows you to add a custom boot image for Graphical LCDs.
+ * With this option Marlin will first show your custom screen followed
+ * by the standard Marlin logo with version number and web URL.
+ *
+ * We encourage you to take advantage of this new feature and we also
+ * respectfully request that you retain the unmodified Marlin boot screen.
+ */
+
+// Show the Marlin bootscreen on startup. ** ENABLE FOR PRODUCTION **
+#define SHOW_BOOTSCREEN
+
+// Show the bitmap in Marlin/_Bootscreen.h on startup.
+//#define SHOW_CUSTOM_BOOTSCREEN
+
+// Show the bitmap in Marlin/_Statusscreen.h on the status screen.
+//#define CUSTOM_STATUS_SCREEN_IMAGE
+
+// @section machine
+
+/**
+ * Select the serial port on the board to use for communication with the host.
+ * This allows the connection of wireless adapters (for instance) to non-default port pins.
+ * Serial port -1 is the USB emulated serial port, if available.
+ * Note: The first serial port (-1 or 0) will always be used by the Arduino bootloader.
+ *
+ * :[-1, 0, 1, 2, 3, 4, 5, 6, 7]
+ */
+#define SERIAL_PORT 3
+
+/**
+ * Select a secondary serial port on the board to use for communication with the host.
+ * :[-1, 0, 1, 2, 3, 4, 5, 6, 7]
+ */
+#define SERIAL_PORT_2 -1
```

Comment:
It fails to build with
```
In file included from Marlin/src/HAL/STM32F1/../../inc/MarlinConfig.h:49:0,
                 from Marlin/src/HAL/STM32F1/HAL.cpp:30:
Marlin/src/HAL/STM32F1/../../inc/SanityCheck.h:2310:6: error: #error "SERIAL_PORT_2 is set to -1, but the MOTHERBOARD has no native USB support. Set SERIAL_PORT_2 to a valid value for your board."
     #error "SERIAL_PORT_2 is set to -1, but the MOTHERBOARD has no native USB support. Set SERIAL_PORT_2 to a valid value for your board."
```

Reply:
Oh sorry for that
please set serial port_2 1

i've ported it from my robin nano V2 board and forget to change this values
thanks

Reply:
Hm, is it should be fixed too?
https://github.com/MarlinFirmware/Configurations/pull/336/files/2d9018d4f8ea85c4387cddcd9ac685725a30a090#diff-a0ec2751395d0460bbd0723f27b06bea0707681b4985cdf2a2bb4cd48352942dR132

Reply:
I believe only this line is incorrect
if you have another issue, please tell me

Reply:
It doesn't boot stacks at `TFT Updating 100%`

Reply:
can be your sd card
original TT SD card is a crap
try to format it and update again

Reply:
I've seen in le3tspeak's fork that "robin_nano"
https://github.com/le3tspeak/Marlin-2.0.X-MKS-Robin-Nano/blob/MKS-Robin-Nano/Marlin/Configuration.h#L372

Reply:
i use my own marlin in sapphire pro
here everything works fine
flash problem may be a sd card faulty

Reply:
I've used 2.0-bugfix and looking through configuration.h I've found some newer changes missing in you PR.

Reply:
Tried new SD card with same result.

Reply:
Florian Marlin works?
did you use latest bugfix to compile config file?

Reply:
I don't know I've stated with your 😌

Reply:
i solved problem
i dont know why my PR isnt update here
now everything will work